src/release.js

```diff
@@ -90,7 +88,7 @@ module.exports = function taoExtensionReleaseFactory(params = {}) {
          * @param {Object} data
          */
         setData(newData) {
-            data = newData;
+            Object.assign(data, newData);
```

Comment:
This function is only used for the tests and I don't see any need to change its behaviour now.